api/xds_protocol.rst

```diff
@@ -60,6 +60,7 @@ correspondence between an xDS API and a resource type. That is:
 -  CDS: :ref:`envoy.api.v2.Cluster <envoy_api_msg_Cluster>`
 -  EDS: :ref:`envoy.api.v2.ClusterLoadAssignment <envoy_api_msg_ClusterLoadAssignment>`
 -  SDS: :ref:`envoy.api.v2.Auth.Secret <envoy_api_msg_Auth.Secret>`
+-  TDS: :ref:`envoy.service.discovery.v2.Runtime <envoy_api_msg_service.discovery.v2.Runtime>`
```

Comment:
Per our offline convo, I think we are running out of useful 3-letter acronyms. It's probably too late to change this to "RuntimeDS", but maybe can we make it more clear in the docs what TDS actually stands for? I keep forgetting.

Reply:
We can also go with > 3 letter DS, e.g. this could be `RTDS`, the TAP one could be `TAPDS` (CC @yuval-k)

Reply:
For reference, @silentdai is working on Filterchain Discovery Service, which will be FCDS.

Reply:
If you are up for changing, I think we should change it. It's pretty confusing. Personally, I think we can go even longer and just use "RuntimeDS" but whatever you think is best.

Reply:
Done